pkgs/data/icons/catppuccin-cursors/default.nix

```diff
@@ -15,58 +13,40 @@ let
   variantName = { palette, color }: (lib.strings.toLower palette) + color;
   variants = map variantName product;
 in
-stdenvNoCC.mkDerivation {
+stdenvNoCC.mkDerivation rec {
   pname = "catppuccin-cursors";
-  version = "unstable-2022-08-23";
+  version = "0.2.0";
+  dontBuild = true;
 
   src = fetchFromGitHub {
     owner = "catppuccin";
     repo = "cursors";
-    rev = "3d3023606939471c45cff7b643bffc5d5d4ff29c";
-    sha256 = "1z9cjxxsj3vrmhsw1k05b31zmncz1ksaswc4r1k3vd2mmpigq1nk";
+    rev = "v${version}";
+    sha256 = "sha256-pz14QY8OMPioQSlXhENPWrxg2rjJ6PrzsCi51e1AxIE=";
   };
```

Comment:
This currently fetches the SVGs twice, once in the ZIPs and once as the actual source files.

Can you use `sparseCheckout` to avoid this? It would be done as `sparseCheckout = [ "cursors" ]`.